### PR TITLE
Enhancement/more html structure in math log

### PR DESCRIPTION
**Pull request by alec-deason**
We have continuing problems with modelers (and me) not being able to read the mathlogs that show up in epiviz. This is an attempt to make them more legible by adding some visual (and markup) structure to them.

Just in case you can't render HTML and CSS in your head and don't want to figure out how to visualize it without EPIViz here's a screenshot of what it would look like:
image

### CODE REVIEW

File: src/cascade/executor/dismod_runner.py [hunk 1]
```diff
@@ -172,14 +172,14 @@ def dismod_report_info(text):
     """This ensures MATHLOG messages have a function name in the log.
     Otherwise, they show <lambda> as the function name.
     """
-    MATHLOG.info(text)
+    MATHLOG.info(text, extra=dict(is_dismod_output=True))
```
Comment (nrhawkins) [inline src/cascade/executor/dismod_runner.py]: function name is nice!

File: src/cascade/executor/epiviz_runner.py [hunk 2]
```diff
@@ -335,13 +340,21 @@ def main(args):
...
 
         if not args.no_upload:
+            MATHLOG.debug(f"Uploading results to epiviz")
             save_model_results(ec)
+        else:
+            MATHLOG.debug(f"Skipping results upload because 'no-upload' was selected")
+    else:
+        MATHLOG.debug(f"Only creating the base db file because 'db-only' was selected")
+
+    MATHLOG.debug(f"Completed succesfully")
```
Comment (nrhawkins) [inline src/cascade/executor/epiviz_runner.py]: succesfully -> successfully

File: src/cascade/executor/math_log.py [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+
+.log_line.error {
+    background: hsl(0, 100%, 79%);
+}
+
+</style>
+"""
+
+INSTANTANIOUSNESS_THRESHOLD = 0.5
+
```
Comment (nrhawkins) [inline src/cascade/executor/math_log.py]: is this: instantaneous?  with an E?

File: src/cascade/executor/math_log.py [hunk 4]
```diff
@@ -0,0 +1,90 @@
...
+
+class MathLogFormatter(Formatter):
+    def __init__(self):
+        datefmt = "%y-%m-%d %H:%M:%S"
+        super().__init__(datefmt=datefmt)
+        self.in_dismod_output = False
+        self.last_event = 0.0
+        self.has_emited_css = False
```
Comment (nrhawkins) [inline src/cascade/executor/math_log.py]: is this "emitted"? with two t's?

File: src/cascade/executor/epiviz_runner.py (hunk 2 above)
Comment (alec-deason) [reply src/cascade/executor/epiviz_runner.py]: thanks

**Pull request merged**